package.json

```diff
@@ -1,25 +1,28 @@
 {
-  "name": "generator-jhipster-entando",
-  "version": "0.0.0",
+  "name": "@entando/component-generator",
```

Comment:
I saw the conversation about the name, just need to make sure Jhipster is able to read the blueprint using the `--blueprint` command

Reply:
Indeed I'm afraid the name needs to stay that way

https://www.jhipster.tech/modules/creating-a-blueprint/

Reply:
Apparently scope is not supported too yet
From yeoman documentations (https://yeoman.io/generators/)

> The name should be generator-your-unique-name. This means it should begin with `generator-` and be followed by a unique (to npm) name. Scoped name support is coming in the future.